autoparser/config.json

```diff
@@ -0,0 +1,47 @@
+{
+  "stopwords": [
+    "a",
+    "an",
+    "the",
+    "if",
+    "for",
+    "has",
+    "at",
+    "of",
+    "and",
+    "or",
+    "then",
+    "treatment",
+    "agent"
+  ],
+  "schemas": {
+    "subject": "../schemas/dev/subject.schema.json",
+    "visit": "../schemas/dev/visit.schema.json",
+    "observation": "../schemas/dev/observation.schema.json"
+  },
+  "column_mappings": {
```

Comment:
are these column mappings from the dictionary files? do we assume they are standard?

Reply:
They will be standard for most REDCap studies, but may need changing, so I have put that in the config